.changelog/16298.txt

```diff
@@ -0,0 +1,3 @@
+```release-note:improvement
```

Comment:
Should this be a feature instead?

Reply:
This was what @cthain used in #15970, adding the same functionality for AWS/GCP. I'm guessing the idea is that it is improving the functionality from the hack support (that they could hardcode the /login data) to real support.

Reply:
Yes, that was my thought process in choosing `improvement` instead of `feature`.